### PR TITLE
snippets: Add example of cryptographic code

### PR DESCRIPTION
**Pull request by dsprenkels**
This PR adds an example of how assembly directives are used in cryptographic code.

### CODE REVIEW

File: snippets/faz-donenfeld-neves_crypto.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+* Authors: Armando Faz, Jason A. Donenfeld, Samuel Neves
+* Reason: side-channel resistance required + performance critical
+* Source: https://github.com/armfazh/rfc7748_precomputed
+* License: [BSD-3-clause](https://github.com/armfazh/rfc7748_precomputed/blob/master/LICENSE)
```
Comment (bjorn3) [inline snippets/faz-donenfeld-neves_crypto.md:5]: Maybe add `* Language: C`, as the rest of the snippets in this repo are written in Rust.

File: snippets/faz-donenfeld-neves_crypto.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+# Code
+
+I did not write this code myself, but I feel this snippet is a good example of
```
Comment (bjorn3) [inline snippets/faz-donenfeld-neves_crypto.md]:
```suggestion
This snippet is probably a good example of
```

Using `I` can be confusing as to who it refers to.

**Review by bjorn3 — APPROVED** [review]
Thanks! I just have a two minor comments.

**Comment by dsprenkels** [comment]
Updated the PR following the comments from @bjorn3.

**Pull request merged**